stubs/pyxdg/xdg/util.pyi

```diff
@@ -0,0 +1,14 @@
+import sys
+from _typeshed import StrPath
+from typing_extensions import Literal, TypeVar
+
+_StrPathT = TypeVar("_StrPathT", bound=StrPath)
+
+PY3: Literal[True]
+
+def u(s: str) -> str: ...
+
+if sys.version_info >= (3, 3):
```

Comment:
We don't care about 3.2, so we can just unconditionally re-export `shutil.which`.